docs/post-actions/customisablePostActions.md

```diff
@@ -6,38 +6,34 @@ This allow you to customise your process, according to the result of a rule.
 You can:
 
 - log informations,
-- send webhook with data priviously proccessed,
+- send webhook with data priviously processed,
 - comment an issue or a PR,
 - and much more!
 
 ## How does it work?
 
 In your `rules.yml` config file, you can add `callback` functions that will be called if the rule succeeds or not.
 
-These `callback`s are `Runnable` classes, that implementing the `RunnableInterface` interface.
-The `Runnable` interface will have a `run()` method with `RuleResult` and `args` as arguments.
+These `callback`s are `Runnable` classes, that implementing the `Runnable` abstract class.
+The `Runnable` interface will have a `run()` method with `callbackType`, `RuleResult` and `args` as arguments.
```

Comment:
`Runnable` is an interface or an abstract class ?

Reply:
Class indeed